### DISCUSSION TITLE
Один раз за всё время

### DISCUSSION THREAD

**Discussion by ItzIsonn** [iBuseWinner/Reputation#18] [Ideas]
Сделать возможность отключать ситуацию: первый дал второму +1, а потом через 2 часа (ну или сколько там ограничение) опять выдал второму +1.
Что-то по типу такого: enable_one_time_vote: true (можно только один раз дать одному и тому же игроку +1/-1)

**Comment by iBuseWinner** [comment]
В оригинальной версии плагина такой функционал добавляться не будет.
Есть возможность создать отдельную версию для вас, стоить это будет 500₽: исправления в дальнейшем - бесплатно, изменение функционала - от 200₽ в зависимости от сложности реализации.
